raiden/transfer/state.py

```diff
@@ -85,9 +85,11 @@ class ChannelState(Enum):
     ChannelState.STATE_SETTLED,
 )
 
-NODE_NETWORK_UNKNOWN = "unknown"
-NODE_NETWORK_UNREACHABLE = "unreachable"
-NODE_NETWORK_REACHABLE = "reachable"
+
+class NetworkState(Enum):
+    NODE_NETWORK_UNKNOWN = "unknown"
```

Comment:
I'd remove the `NODE_NETWORK_` prefix from all of them

Reply:
+1, no reason to namespace it anymore if it's part of an enum